scripts/lint-samples.js

```diff
@@ -0,0 +1,44 @@
+const { execSync } = require('child_process');
+const samples = require('./samples.json');
+
+/**
+ * Start linting process only for the samples that were affected by the new changes in:
+ * - create-sitecore-jss/src/templates/**
+ * - create-sitecore-jss/src/initializers/**
+ */
+
+const affectedTemplates = execSync('git diff --name-only packages/create-sitecore-jss', {
```

Comment:
Do we need to explicitly specify the target branch here?

Reply:
Valid point, I also updated our internal script to replace _origin/dev_ during the release prep